docs/user/rss/flow/input/input.md

```diff
@@ -0,0 +1,225 @@
+
+```yaml
+# General Parameters
+tag: 'Si'
+train_from_scratch: false
+resume_from_previous_state:
+  test_error:
+  pre_database_dir:
+  mlip_path:
+  isolated_atom_energies:
+```
+
+The `tag` identifies the elements in the system, such as `Si` in this example, and can also be passed to the parameters for random structure generation. Setting `train_from_scratch=false` indicates that the RSS workflow will start from scratch. To resume a workflow, refer to the [Resuming workflow from point of interruption](../quick_start/start.md) section.
+
+## Buildcell parameters
+
+The section defines the settings for generating initial random structures. These parameters control the diversity, symmetry, size, and number of the generated structures. We utilize the `buildcell` software from [AIRSS](https://airss-docs.github.io/technical-reference/buildcell-manual) for random structure generation, ensuring flexibility and efficiency in exploring configurational space.
+
+```yaml
+# Buildcell Parameters
+generated_struct_numbers:
+  - 8000
+  - 2000
+buildcell_options:
+  - NFORM: '1'
+    SYMMOPS: '1-4'
+    SLACK: 0.25
+    OVERLAP: 0.1
+    NATOM: '{6,8,10,12,14,16,18,20,22,24}'
+  - NFORM: '1'
+    SYMMOPS: '1-4'
+    SLACK: 0.25
+    OVERLAP: 0.1
+    NATOM: '{7,9,11,13,15,17,19,21,23}'
+fragment_file: null
+fragment_numbers: null
+num_processes_buildcell: 128
+```
+
+The `buildcell_options` parameter is one of the most critical settings, as it determines the scope of the RSS search and directly influences the diversity of the generated structures. We provide multiple sets of `buildcell` input parameters. In the configuration above, 8,000 even-numbered structures and 2,000 odd-numbered structures will be generated, with symmetry operations ranging from 1 to 4. You can supply a single set or multiple sets as needed. This flexibility ensures that the initial structures have sufficient diversity. In principle, any parameter supported by [buildcell](https://airss-docs.github.io/technical-reference/buildcell-manual) can be used in the `buildcell_options` section.
+
+The `fragment_file` and `fragment_numbers` parameters are used during random structure generation to define specific fragments as the smallest building blocks. For example, you can define an H2O molecule as a fragment and use it as the basic unit for generating random structures. This allows for more customized and realistic initial configurations when working with molecular or other complex systems. The `num_processes_buildcell` parameter specifies the number of CPU cores to be used in parallel during random structure generation. Note that this parameter is limited to a single node.
+
+> **Note**: The `generated_struct_numbers` and `buildcell_options` parameters must have the same length. Each entry in `buildcell_options` corresponds to the number of structures specified at the same position in `generated_struct_numbers`.
```

Comment:
It means the `buildcell_options` dict needs to be repeated like 10000 times right?

Reply:
Yes. It is very fast. I think this process only takes a few minutes.